python/cross_service/aurora_item_tracker/README.md

```diff
@@ -118,81 +149,204 @@ credentials, such as `arn:aws:secretsmanager:us-west-2:123456789012:secret:docex
-sandbox, you must register both the sender and recipient email addresses. You can
-do this by using [the console](https://console.aws.amazon.com/ses).
+#### Run the service
 
-*Tip:* For this example, you can register the same email account for both the sender and 
-the recipient.
-
-#### Running the web server
+This example uses [Flask](https://flask.palletsprojects.com/en/2.0.x/) to host a local 
+web server and REST service. With the web server running, you can send HTTP requests to
+the service endpoint to list, add, and update work items and to send email reports.
 
-Start a local web server in debug mode by running the following at a command prompt:
+Run the app at a command prompt to start the Flask web server. Specify the `--debug`
+flag for more detailed output during development, and specify a port of 8080 to work
+with the Elwing client.
 
 ```
-python app.py
+flask --debug run -p 8080
 ```
 
-This starts the Flask web server in debug mode on http://localhost:5000.
-
 ### Webpage
 
-The REST service is intended to be used in conjunction with the 
-[Work item tracker web client](../../../resources/clients/react/item-tracker).
-The web client is built with React and sends HTTP requests to the REST service.
+The REST service is designed to work with the item tracker plugin in the Elwing web
+client. The item tracker plugin is a JavaScript application that lets you manage work 
+items, send requests to the REST service, and see the results.
 
-Follow the instructions in the [README](../../../resources/clients/react/item-tracker/README.md)
-to install the necessary prerequisites for the client. Configure the endpoint
-to send requests to the local web server at http://localhost:5000.
+#### Run Elwing and select the item tracker
 
-Start the webpage in development mode by running the following at a command prompt
-in the [Work item tracker web client](../../../resources/clients/react/item-tracker) folder:
+1. Run Elwing by following the instructions in the [Elwing README](/resources/clients/react/elwing/README.md).
+1. When Elwing starts, a web browser opens and browses to http://localhost:3000/.
+1. Run the item tracker plugin by selecting **Item Tracker** in the left navigation bar.
+1. This sends a request to the REST service to get any existing active items:
+   ```
+   GET http://localhost:8080/api/items?archived=false
+   ```
+1. At first, the table is empty.
 
-```
-npm start
-```
+    ![Work item tracker](images/item-tracker-start.png)
 
-This compiles the project and browses to it at http://localhost:3000.
+1. Select **Add item**, fill in the values, and select **Add** to add an item.
 
-### Using the app
+    ![Add item](images/item-tracker-add-item.png)
 
-When you first start, the application shows that your database contains no work items.
+   This sends a POST request to the REST service with a JSON payload that contains the
+   work item.
 
----
+   ```
+   POST http://localhost:8080/api/items
```

Comment:
While I appreciate this level of detail, would it be less redundant to limit Elwing instructions to the plugin pages?

Reply:
Agreed, but I'm going to keep this here until we've established how that's going to work.